spec/support/form_helpers.rb

```diff
@@ -337,6 +346,7 @@ def fill_in_activity_form(
         expect(page).to have_content policy_marker_nutrition
       end
     end
+    expect(page).to have_content fund_pillar if parent&.roda_identifier_compound&.include?("NF")
```

Comment:
This is risky - should be using`parent&.associated_fund&.roda_identifier_fragment == "NF"`

If I created a GCRF-funded programme whose `roda_identifier_fragment` was, for example, `INFO01` this would pass.

Reply:
Oh, good call